features/bevragen/persoon/verblijfplaats/dev/autorisatie-gba.feature

```diff
@@ -137,21 +137,25 @@ Functionaliteit: autorisatie verblijfplaatsgegevens Persoon
       | datumIngangGeldigheid            | 10120 80910 81010 81030 81110 81150 81160 81210 81310 81320 81330 81340 81350 | 88510                |
 
 
-  Rule: Een gemeente als afnemer is geautoriseerd voor alle verblijfplaatsgegevens van eigen inwoners
-
+  Rule: Een gemeente als afnemer is geautoriseerd voor alle verblijfplaats gegevens van eigen inwoners
+    Wanneer de afnemer parameter gemeenteVanInschrijving gebruikt 
+     en die is gelijk aan de waarde van gemeenteCode in de 'claim', 
+     dan wordt niet gekeken naar de autorisatie van de afnemer
+     
     Scenario: Gemeente vraagt van een eigen inwoner om velden in verblijfplaats en er zit geen enkel verblijfplaatsgegeven in de autorisatie
       Gegeven de afnemer met indicatie '000008' heeft de volgende 'autorisatie' gegevens
       | Rubrieknummer ad hoc (35.95.60) | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
       | 10120                           | N                        | 20201128                |
       En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
       | naam         | waarde |
       | afnemerID    | 000008 |
-      | gemeenteCode | 0599   |
+      | gemeenteCode | 0800   |
       Als gba personen wordt gezocht met de volgende parameters
-      | naam                | waarde                                                                               |
-      | type                | RaadpleegMetBurgerservicenummer                                                      |
-      | burgerservicenummer | 000000024                                                                            |
-      | fields              | verblijfplaats.verblijfadres.korteStraatnaam,verblijfplaats.verblijfadres.huisnummer |
+      | naam                    | waarde                                                                               |
+      | type                    | RaadpleegMetBurgerservicenummer                                                      |
+      | burgerservicenummer     | 000000024                                                                            |
+      | gemeenteVanInschrijving | 0800                                                                                 |
```

Comment:
wat is de response als de gevraagde persoon niet in de opgegeven gemeente staat ingeschreven?

Reply:
dan vind je 0 personen